datafusion/core/tests/sql/timestamp.rs

```diff
@@ -579,8 +579,8 @@ async fn timestamp_add_interval_months() -> Result<()> {
     let t1_naive = chrono::NaiveDateTime::parse_from_str(res1, format).unwrap();
     let t2_naive = chrono::NaiveDateTime::parse_from_str(res2, format).unwrap();
 
-    let year = t1_naive.year() + (t1_naive.month() as i32 + 17) / 12;
-    let month = (t1_naive.month() + 17) % 12;
+    let year = t1_naive.year() + (t1_naive.month0() as i32 + 17) / 12;
```

Comment:
Instead of using `NOW()` we could use some table of dates instead to test it?.

Reply:
This would avoid the test breaking suddenly on a certain date.

Reply:
@Dandandan It seems your suggestion is great, it failed today again :(